command-expansion-server/src/lib/codegen/CommandEDSLPreface.ts

```diff
@@ -39,17 +39,42 @@ export class Command<A extends ArgType[] | { [argName: string]: any } = [] | {}>
     }
   }
 
+  private buildSeqTime(time: Temporal.Instant | Temporal.Duration): string {
+    // return  yyyy-doyThh:mm:ss.sss
+    if (time instanceof Temporal.Instant) {
+      const utcZonedDate = time.toZonedDateTimeISO('UTC');
+      return `${utcZonedDate.year}-${this.format(utcZonedDate.dayOfYear, 3)}T${this.format(
+        utcZonedDate.hour,
+        2,
+      )}:${this.format(utcZonedDate.minute, 2)}:${this.format(utcZonedDate.second, 2)}.${this.format(
+        utcZonedDate.millisecond,
+        3,
+      )}`;
+    }
+    // return hh:mm:ss.sss
+    else {
+      return `${this.format(time.hours, 2)}:${this.format(time.minutes, 2)}:${this.format(
+        time.seconds,
+        2,
+      )}.${this.format(time.milliseconds, 3)}`;
+    }
+  }
+
+  private format(number: number, size: number = 2): string {
```

Comment:
I'm curious about the default size - why `2`? I would vote for not having a default size.

Reply:
Good point, I will remove it cause I am overriding the default anyways when calling this method.